src/sentry/integrations/utils/code_mapping.py

```diff
@@ -16,6 +16,14 @@ class CodeMapping(NamedTuple):
     source_path: str
 
 
+class CodeMappingMatch(NamedTuple):
```

Comment:
This helps us pass all the necessary info to identify and create code mappings

Reply:
I think the whole PR would be better by removing `CodeMappingMatch` completely and just using `CodeMapping`.

Are you okay to give it a try?